esi_leap/resource_objects/test_node.py

```diff
@@ -42,8 +42,17 @@ def get_lease_uuid(self):
     def get_lessee_project_id(self):
         return self._project_id
 
+    def get_node_power_state(self):
+        return 'Off'
+
+    def get_node_provision_state(self):
+        return 'available'
+
```

Comment:
Should these methods be added to dummy nodes as well?

Reply:
Added. Also added unit test in test_dummy_nodes.py